lib/msal-browser/src/cache/BrowserStorage.ts

```diff
@@ -254,19 +284,37 @@ export class BrowserStorage implements ICacheStorage {
     }
 
     /**
-     * Dummy implementation until browser cache is migrated
+     * Gets the cache object referenced by the browser
      */
     getCache(): object {
         return this.windowStorage;
     }
 
     /**
-     * Dummy implementation until browser cache is migrated
+     * interface compat, we cannot overwrite browser cache; Functionality is supported by individual entities in browser
      */
     setCache(): void {
         // sets nothing
     }
 
+    /**
+     * Prepend msal.<client-id> to each key; Skip for any JSON object as Key (defined schemas do not need the key appended: AccessToken Keys or the upcoming schema)
+     * @param key
+     * @param addInstanceId
+     */
+    generateCacheKey(key: string): string {
+        try {
+            // Defined schemas do not need the key migrated
+            this.validateObjectKey(key);
```

Comment:
Outside the scope of this PR, but could `validateObjectJey` just return a boolean, since you are catching and handling all errors?

Reply:
Done in a future PR.